.github/workflows/scripts/slack_gem_notifications/cve_methods.rb

```diff
@@ -0,0 +1,32 @@
+# encoding: utf-8
+# This file is distributed under New Relic's license terms.
+# See https://github.com/newrelic/newrelic-ruby-agent/blob/main/LICENSE for complete details.
+# frozen_string_literal: true
+
+require 'time'
+require 'feedjira'
+require 'httparty'
+
+# limiting to 5 checks since more than 5 security concerns in 24 hours is unlikely. 
+def check_for_cves
+  xml = HTTParty.get('https://rubysec.com/atom.xml').body
+  feed = Feedjira.parse(xml)
+  feed.entries.take(5).each do |entry|
+    if Time.now.utc - entry.updated < 24 * 60 * 60
```

Comment:
The cool thing about the RubySec feed in particular is that it is sorted in reverse chronological order (giving you the newest entry first). As a result, we should be able to do a `break` to stop processing any more feeds as soon as we find one older than the timeframe, rather than testing every single feed entry for being recent.

Reply:
A much better way to do it! [Addressed here](https://github.com/newrelic/newrelic-ruby-agent/pull/1292/commits/af0ee976fa859d6338e6fb22391749df36f714ce).